app/views/about.html

```diff
@@ -1 +1,43 @@
-<p>This is the about view.</p>
+<div class="about">
+    <h1>Welcome to MoodCat.me</h1>
+    <p>
+        Hi there!
+        Welcome to MoodCat.me, the music platform of the future, available today.
+        Join us in the adventure of finding and enjoying music geared to you!
+        We try to understand you
+    </p>
+    <h2>How does it work?</h2>
+    <p>
+        Go to the homepage, and select what mood(s) you're in.
+        Once you click next, we recommend you a couple of rooms to check out.
+        Pick one you think you like, and you will join the room.
```

Comment:
'Pick the one you like to join the room.' it sounds more direct and clear for the user.